lib/dialects/postgres/schema/compiler.js

```diff
@@ -76,12 +76,20 @@ SchemaCompiler_PG.prototype.createSchemaIfNotExists = function(schemaName) {
   );
 };
 
-SchemaCompiler_PG.prototype.dropSchema = function(schemaName) {
-  this.pushQuery(`drop schema ${this.formatter.wrap(schemaName)}`);
+SchemaCompiler_PG.prototype.dropSchema = function(schemaName, cascade = false) {
+  this.pushQuery(
+    `drop schema ${this.formatter.wrap(schemaName)}${cascade && ' CASCADE'}`
+  );
 };
 
-SchemaCompiler_PG.prototype.dropSchemaIfExists = function(schemaName) {
-  this.pushQuery(`drop schema if exists ${this.formatter.wrap(schemaName)}`);
+SchemaCompiler_PG.prototype.dropSchemaIfExists = function(
+  schemaName,
+  cascade = false
+) {
+  this.pushQuery(
+    `drop schema if exists ${this.formatter.wrap(schemaName)}${cascade &&
```

Comment:
This would evaluate to `drop schema if exists <tablename>false` when cascade is false.

Please change to a ternary-operation.

```javascript
drop schema if exists ${this.formatter.wrap(schemaName)}${cascade ? ' CASCADE' : ''}
```